### PR TITLE
Add internal endpoints for embed

### PR DESCRIPTION
**Pull request by sreehari-aot**
Deprecate `/token` endpoint
Deprecate `/application/external/create`
Add internal, external endpoints for fetching
and submitting embedd forms

# Issue Tracking

JIRA: 
Issue Type: BUG/ FEATURE

# Changes

# How has the change been tested?

# Screenshots (if applicable)

# Build Success screenshot (Till a CICD pipeline is set up)

# Notes

### CODE REVIEW

File: forms-flow-api/src/formsflow_api/resources/form_embed.py [hunk 1]
```diff
@@ -121,28 +163,16 @@ def post():
...
-            return response, status
-        except BusinessException as err:
-            return err.error, err.status_code
+        return EmbeddCommonMethods.post()
 
 
 @cors_preflight("GET,OPTIONS")
-@API.route("/form/<string:path>", methods=["GET", "OPTIONS"])
-class Form(Resource):
+@API.route("/internal/form/<string:path>", methods=["GET", "OPTIONS"])
```
Comment (shuhaib-aot) [inline forms-flow-api/src/formsflow_api/resources/form_embed.py:170]: internal ?

File: forms-flow-api/src/formsflow_api/resources/form_embed.py [hunk 2]
```diff
@@ -121,28 +163,16 @@ def post():
...
-            (
-                response,
-                status,
-            ) = CombineFormAndApplicationCreate.application_create_with_submission(
-                data, formio_url, web_url, request.headers["Authorization"]
-            )
-            return response, status
-        except BusinessException as err:
-            return err.error, err.status_code
+        return EmbeddCommonMethods.post()
```
Comment (shuhaib-aot) [inline forms-flow-api/src/formsflow_api/resources/form_embed.py:166]: Embed

File: forms-flow-api/src/formsflow_api/resources/form_embed.py (hunk 1 above)
Comment (sreehari-aot) [reply forms-flow-api/src/formsflow_api/resources/form_embed.py:170]: 2 endpoints for form embedding `/internal` for keycloak authentication `/external` for custom authentication

File: forms-flow-api/src/formsflow_api/resources/form_embed.py (hunk 2 above)
Comment (sreehari-aot) [reply forms-flow-api/src/formsflow_api/resources/form_embed.py:166]: 👍

**Review by abilpraju-aot — APPROVED** [review]
LGTM

**Review by AbijithS-aot — APPROVED** [review]

**Pull request merged**